bin/promote.py

```diff
@@ -68,6 +58,56 @@ def request_reviwers(pull):
 
     print('{0} members were been assigned to #{1}'.format(len(members), pull['url']))
 
+def add_pull_comment(pull_number):
+    try:
+        endpoint_teams = '{0}/orgs/greenpeace/teams/{1}'.format(
+            GITHUB_API,
+            'planet-4-designers'
+        )
+        response_teams = requests.get(endpoint_teams, headers=HEADERS)
+        endpoint_members = response_teams.json()['members_url']
+        endpoint_members = endpoint_members.replace('{/member}', '')
+        response_members = requests.get(endpoint_members, headers=HEADERS)
+
+        members = ''
+        for member in response_members.json():
+            if member['login'] != 'planet-4':
+                members += ' @{0}'.format(member['login'])
+        message = 'Hey{0}, a new Design Tokens version has been promoted to the master theme.\nPlease check after the test it is ready :rocket:.'.format(members)
```

Comment:
Can't we use `@greenpeace/planet-4-designers` directly here to simplify the code?

And potentially same thing can be done in the `request_reviwers` function

Reply:
Let me try with @greenpeace/planet-4-designers

Reply:
About the `request_reviwers` I didn't want to add the `Planet 4 bot` as a reviewer. What do you think?

Reply:
Yes, but I was thinking whether the API allows you to add a team as a reviewer instead of individual members.

Looking at the [documentation](https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request), using `team_reviewers` instead of `reviewers` should work.